beetsplug/subsonicupdate.py

```diff
@@ -90,10 +49,51 @@ def __init__(self):
         config['subsonic']['pass'].redact = True
         self.register_listener('import', self.start_scan)
 
+    @staticmethod
+    def __create_token():
+        """Create salt and token from given password.
+
+        :return: The generated salt and hashed token
+        """
+        password = config['subsonic']['pass'].as_str()
+
+        # Pick the random sequence and salt the password
+        r = string.ascii_letters + string.digits
+        salt = "".join([random.choice(r) for _ in range(6)])
+        salted_password = password + salt
+        token = hashlib.md5(salted_password.encode('utf-8')).hexdigest()
```

Comment:
Also changed this, but hard to tell in GitHub diff...

Used to be `hashlib.md5().update(salted_password.encode('utf-8')).hexdigest()`. Seems like a proper use of `hashlib`.